src/cpu/flexcpu/simple_dataflow_cpu.hh

```diff
@@ -300,16 +334,10 @@ class SimpleDataflowCPU : public BaseCPU
      *  function returns false, the requester should not expect callback_func
      *  to be called in the future.
      */
-    bool requestDataAddrTranslation(const RequestPtr& req, ThreadContext* tc,
+    void requestDataAddrTranslation(const RequestPtr& req, ThreadContext* tc,
                                     bool write,
                                     TranslationCallback callback_func);
 
-    // TODO maybe we just need a generic requestInstSquash function to squash
```

Comment:
This probably shouldn't be in this patch.

Reply:
Hmm felt too small for its own PR though... but the next thing I'm working on is this very issue anyway so yea let's drop it